src/FileManipulator.php

```diff
@@ -113,6 +121,22 @@ public function createTempDirectory() : string
         return $tempDirectory;
     }
 
+    protected function extractVideoThumbnail(string $videoFile, Conversion $conversion) : string
+    {
+        $imageFile = string($videoFile)->pop('.').'.jpg';
```

Comment:
It would be more readable to use `pathinfo` with the `PATHINFO_FILENAME` constant instead of popping.